kubeflow/trainer/rhai/traininghub.py

```
@@ -481,6 +528,42 @@ def _transform_sft(self, metrics):
+
+            LoRA metrics format (from training_metrics.jsonl):
+                {"step": 1, "epoch": 0.015625, "loss": 4.2727, "learning_rate": 2e-6,
+                 "max_steps": 100}
+            """
+            step = metrics.get("step", 0)
+            epoch = metrics.get("epoch", 0)
+            max_steps = metrics.get("max_steps", 0)
+
+            # Calculate progress based on steps, clamping to 99 until final step
+            progress = (step / max_steps * 100) if max_steps > 0 else 0
+            percent_int = int(round(progress))
```

Ruff (0.15.7)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)